### PR TITLE
Performance improvements for start page

### PR DESCRIPTION
**Pull request by denar90**
**TL;DR;**

I noticed that our main page is loading in something strange way. Firstly loaded content with css but it all look broken, then loaded normal layout with fancy styles. I made some video to show it (it's with `regular 2g network`, to show it in 'slow motion')
https://monosnap.com/file/P5uuJQIVHXAsBzKnD3CTb7K5q9EbEY

I dig deeper and made some perf tests. So lets go.
First was analyzed `http://marionettejs.com/`

marionette js the backbone framework 2016-10-22 15-16-58

I cloned our site repo and run dev to debug why it is happening. I was really surprised that after running on dev it looks fine but different in scope of css loading.

marionette js the backbone framework 2016-10-22 15-31-28

As you can see there are blocking css, rendering hasn't started because of it, hence no broken html.

I went deeper and run `compile-all` (this is grunt command which is running during deploying). And
marionette js the backbone framework 2016-10-22 15-40-37

(it was run on 2G network because localhost to fast to show broken html)

So. dev and prod are different. And I got 3 questions.

**Why?** 

It's because we are using `grunt-critical` ONLY for prod.

---

**What to do?**

I propose to get rid of `grunt-critical` at all. Let main css be render blocking then we will not have any broken html which user see coming on our main page.

---

**What influence will it have on perf?**  

After adding missed `defer`, moving none critical css at the bottom and making main css be render blocking I got these results.
marionette js the backbone framework 2016-10-22 15-12-42

So now about perf. Unfortunately render blocking css is not so good and perf results...

I used fancy tool [pwmetrics](https://github.com/paulirish/pwmetrics).

**Current site**

1 bash 2016-10-22 15-08-48

**Old dev results**

1 bash 2016-10-22 15-35-13

**New dev results**

1 bash 2016-10-22 15-09-57

For prod I made double results...

**Old prod results** 

1 bash 2016-10-22 15-45-26

**New prod results**

1 bash 2016-10-22 15-23-42

So I think this perf results can be acceptable for better user experience...

P.S. I was inspired by @samccone and @paulirish [video](https://www.youtube.com/watch?v=6m_E-mC0y3Y)

### CODE REVIEW

**Comment by peterblazejewicz** [comment]
Artem, the idea behind `critical` plugin usage was to correct FOUC without much manual work and coding required. So that went with Grunt plugin via @samccone : https://github.com/marionettejs/marionettejs.com/commit/fc2b881dbd59ca9291de1af12cfa8d8ba31c7be8 as (last) part of effort to improve website scores and visibility.
As the tools are changing this would be better to swap implementation - this would help to improve the website speed (and again, gain better visibility). Here you can see some data from the same period in the past: http://www.webpagetest.org/result/161022_V8_HQ7/
I'd ditch `critical` for now so you could do other optimization first (like correct order of loading resources and which are critical to web site first usage).

**Comment by denar90** [comment]
@peterblazejewicz I also think https://github.com/marionettejs/marionettejs.com/pull/486 can help in future avoid perf decreasing.

**Comment by denar90** [comment]
Can we merge this? @peterblazejewicz @samccone do you have any suggestions?

**Comment by peterblazejewicz** [comment]
@samccone
